### PR TITLE
Comp - 20:  Inset video

### PR DESCRIPTION
**Pull request by codeuri**
- added dynamic sizes
- worked on responsible 
- added stories 
- Unit test for VideoEmbed component
- Unit test for FigureWrap component

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit 794574ffbd65f409bfd56d8ee51eac3fc0a3fc82

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/5c8ad12be0c596000c4ccfdc

**Comment by TimMansell** [comment]
@gepifanio, I think it should be in the embed section.

Do you mean changing the storybook embed section name to Video Embed?  There are soundcloud embeds in that section to so a general embed is fine in that case.

**Comment by TimMansell** [comment]
Also, I don't think we require a small option. @jaredh-um can you confirm.

**Comment by jaredh-um** [comment]
@TimMansell Yeah, I wouldn't bother with anything like the small or medium sizes - too small to view. Large only

File: components/embed/index.css [hunk 1]
```diff
@@ -49,4 +49,43 @@
       height: 20rem;
     }
   }
+
+  &--inset {
+    iframe {
```
Comment (TimMansell) [inline components/embed/index.css]: Don't think we need lines 54-66

**Pull request merged**